Notifications crashes with NPE when simperium is not configured

WordPress built without a Simperium application id will crash whenever the Notifications list is loaded.

```
    java.lang.RuntimeException: Unable to resume activity {org.wordpress.android/org.wordpress.android.ui.notifications.NotificationsActivity}: java.lang.NullPointerException
            at android.app.ActivityThread.performResumeActivity(ActivityThread.java:2742)
            at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:2771)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2235)
            at android.app.ActivityThread.access$600(ActivityThread.java:141)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1234)
            at android.os.Handler.dispatchMessage(Handler.java:99)
            at android.os.Looper.loop(Looper.java:137)
            at android.app.ActivityThread.main(ActivityThread.java:5041)
            at java.lang.reflect.Method.invokeNative(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:511)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:793)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:560)
            at dalvik.system.NativeStart.main(Native Method)
     Caused by: java.lang.NullPointerException
            at org.wordpress.android.ui.notifications.NotificationsListFragment.onResume(NotificationsListFragment.java:85)
            at android.app.Fragment.performResume(Fragment.java:1738)
            at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:919)
            at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1057)
            at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1039)
            at android.app.FragmentManagerImpl.dispatchResume(FragmentManager.java:1850)
            at android.app.Activity.performResume(Activity.java:5192)
            at android.app.ActivityThread.performResumeActivity(ActivityThread.java:2732)
            at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:2771)
            at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2235)
            at android.app.ActivityThread.access$600(ActivityThread.java:141)
            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1234)
            at android.os.Handler.dispatchMessage(Handler.java:99)
            at android.os.Looper.loop(Looper.java:137)
            at android.app.ActivityThread.main(ActivityThread.java:5041)
            at java.lang.reflect.Method.invokeNative(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:511)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:793)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:560)
            at dalvik.system.NativeStart.main(Native Method)
```

--- a/WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java
@@ -82,13 +82,13 @@ public class NotificationsListFragment extends ListFragment implements Bucket.Li
         refreshNotes();
 
         // start listening to bucket change events
-        mBucket.addListener(this);
+        if (mBucket != null) mBucket.addListener(this);
     }
 
     @Override
     public void onPause() {
         // unregister the listener
-        mBucket.removeListener(this);
+       if (mBucket != null) mBucket.removeListener(this);
 
         super.onPause();
     }

